spa/css.stub.js

```diff
@@ -0,0 +1 @@
+module.exports = {};
```

Comment:
In some react component we use `import xxx.css`. 
Seems the `OrgsContainer` we import ScrollBar which import SimpleBar which import `xx.css` and ts-jest is failing on recongise it. 
Search web give me answer that to put a stub here just to pase the ts check.